src/m365/spfx/commands/project/project-upgrade/upgrade-1.22.0.ts

```diff
@@ -144,9 +144,9 @@ export default [
     "includeGlobs": ["webparts/*/loc/*.js"]
   }
 }
-`),
-  new FN020001_RES_types_react('17.0.45'),
-  new FN021001_PKG_main(false),
+` }),
+  new FN020001_RES_types_react({ packageVersion: '17.0.45' }),
+  new FN021001_PKG_main({ add: false }),
   new FN021004_PKG_scripts_build('heft test --clean --production && heft package-solution --production'),
```

Comment:
Seems like we still have rules that use separate args instead of options. Update